src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplIntegrationTest.java

```diff
@@ -123,5 +165,23 @@ void testNumber() {
         float returnedNumber = awsSecretServiceImpl.getSingleNumberValueFromAwsSecret(SINGLE_FLOAT_SECRET_KEY_NAME, true);
         assertEquals(floatNumber, returnedNumber);
     }
+    
+    @Test
+    void testRetrieveSecretFromCacheNull() {
+        awsSecretServiceImpl.clearCache();
+        
+        Exception exception = assertThrows(RuntimeException.class, () -> {
+            awsSecretServiceImpl.retrieveSecretFromCache(null);
+        });
+
+        String actualMessage = exception.getMessage();
```

Comment:
This message is not being used anywhere. Can this line be removed if the test doesn't need to do anything with the message?

Also, if this line gets removed, you could simplify the assertThrows() call to not store the returned exception in a variable.

Reply:
actually, I had a string comparison assertion that I must have accidentally removed, I shall put it back.